Formula/pnpm.rb

```diff
@@ -20,14 +20,66 @@ class Pnpm < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "52d1cbcba6a433d4f18eef3d3efafe44ac46b41fcbbc6be1956109ce69cb305f"
   end
 
-  depends_on "node"
+  depends_on "node@14" => :build
```

Comment:
Why are we downgrading the node version?

Reply:
Because the official binary uses node 14. I can't guarantee but I think node 16 is OK.